INSTALL

```diff
@@ -393,6 +393,11 @@
                    go in libssl (Note: this is a different capability to the
                    pipelining functionality).
 
+  newversion-only
+                   Don't build with pre-3.0.0 version scheme macros, i.e.
+                   remove support for the macros OPENSSL_VERSION_NUMBER and
+                   OPENSSL_VERSION_TEXT.
```

Comment:
Why do we need a new option for this? Who will use this?

Reply:
Oh, that doc needs updating.

> Who will use this?

Anyone that wants to try with the new macros only.

Reply:
I'm not sure why someone would want to do that. It doesn't seem like a compelling reason to add a new config option - we've got too many of those as it is. I think we should remove this.

Reply:
Doc updated

Reply:
Hmm, I guess that `no-deprecated` or `--api=3.0.0` should suffice, so ok